ext/nnstreamer/tensor_decoder/nnstreamer.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
```

Comment:
Tensor_converter needs this schema file.
How about move the file to `ext/nnstreamer/include`.
And generated file also.

Reply:
I think you're right. The suggestion is applied.